sources/academy/platform/get_most_of_actors/actor_readme.md

```diff
@@ -5,93 +5,96 @@ sidebar_position: 2
+Our scrapers are ethical and do not extract any private user data, such as email addresses, gender, or location. They only extract what the user has chosen to share publicly. We therefore believe that our scrapers, when used for ethical purposes by Apify users, are safe. However, you should be aware that your results could contain personal data. Personal data is protected by the [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) in the European Union and by other regulations around the world. You should not scrape personal data unless you have a legitimate reason to do so. If you're unsure whether your reason is legitimate, consult your lawyers. You can also read our blog post on the [legality of web scraping](https://blog.apify.com/is-web-scraping-legal/)
 
-    ![How to scrape a website - numbered tutorial](./images/how-to-scrape-target-site.webp)
+:::
 
-5. **Is it legal to scrape (target site)?**
+### Input parameters
 
-    - This can be used as a boilerplate text for the legal section, but you should use your own judgement and also customize it with the site name.
+Refer to the Actor's input tab on the detail page. You can include a screenshot showing the input fields for better clarity.
 
-    > Our scrapers are ethical and do not extract any private user data, such as email addresses, gender, or location. They only extract what the user has chosen to share publicly. We therefore believe that our scrapers, when used for ethical purposes by Apify users, are safe. However, you should be aware that your results could contain personal data. Personal data is protected by the [GDPR](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation) in the European Union and by other regulations around the world. You should not scrape personal data unless you have a legitimate reason to do so. If you're unsure whether your reason is legitimate, consult your lawyers. You can also read our blog post on the [legality of web scraping](https://blog.apify.com/is-web-scraping-legal/)
+:::info Example of input schema
 
-6. **Input**
+Twitter Scraper has the following input options. Click on the [input tab](https://apify.com/quacker/twitter-scraper/input-schema) for more information.
 
-    - Refer to the input tab on Actor's detail page. If you like, you can add a screenshot showing the user what the input fields will look like.
-    - This is an example of how to refer to the input tab:
+:::
 
-    > Twitter Scraper has the following input options. Click on the [input tab](https://apify.com/quacker/twitter-scraper/input-schema) for more information.
+### Output Data
 
-7. **Output**
+Mention that the extracted dataset can be downloaded in various formats such as `JSON`, `HTML`,or `CSV`. Provide a simplified `JSON` dataset example to illustrate the output structure, check [this Actor](https://apify.com/compass/crawler-google-places#output-example) for an example.
```

Comment:
There's a missing whitespace after the comma, but when suggesting that, I realized I wouldn't treat the abbreviations as code blocks. These are very common abbreviations in the programming world and I wouldn't think they deserve such emphasis.

```suggestion
Mention that the extracted dataset can be downloaded in various formats such as JSON, HTML, or CSV. Provide a simplified JSON dataset example to illustrate the output structure, check [this Actor](https://apify.com/compass/crawler-google-places#output-example) for an example.
```

Reply:
After some consideration, that makes sense and I agree